content/millennium/r4/diagnostic/observation.md

```diff
@@ -246,15 +246,19 @@ Update an existing Observation.
 
 _Implementation Notes_
 
-* Currently only `laboratory` is supported.
+* Currently only `laboratory` and `vital-signs` is supported.
```

Comment:
change 'is' to 'are'

Reply:
[commit](https://github.com/cerner/fhir.cerner.com/pull/675/commits/79d5eab8c4b005269538ab07cfc833058b9698f5#diff-fbd7021b360068e6d7f7f8ef265148e4572c35c1766f37c4280d8833b3c24749R249)